Websites/webkit.org/wp-content/themes/webkit/widgets/twitter.php

```diff
@@ -14,7 +14,7 @@ class WebKitTwitterTileWidget extends WebKitPostTileWidget {
     const ENDPOINT = 'https://api.twitter.com/1.1/statuses/user_timeline.json';
 
     function __construct() {
-        parent::WP_Widget(false,
+        parent::__construct(false,
```

Comment:
My understanding is this will invoke constructor of `WebKitPostTileWidget`, which does not take any argument in its constructor. In this case, arguments will be ignored.